Hexapod/v1/default.yaml

```diff
@@ -0,0 +1,42 @@
+camera_config:
+  elevation_coeffs:
+    # Fit of https://github.com/bxin/hexrot/blob/master/LUT/Camera%20Hexapod%20Motions%20in%20Elevation%20Axis%202020%2007%2023.xlsx
+    # commit 0513b27 2020-08-04
+    - [0]
+    - [6.41754532e+02, 5.52880420e-01, -2.06081416e-01, 1.14135938e-04, 3.68377168e-06]
+    - [-6.05001038e+02, 1.58266699e+01, 4.76907398e-03, -1.00806469e-03, 2.63260121e-06]
+    - [1.77291858e-02, -2.16979613e-07, -5.39973300e-06, -1.50534189e-10, 1.41927682e-10, -7.00491997e-14, -9.63132987e-16]
+    - [0]
+    - [0]
+  temperature_coeffs:
+    # From Document 7334 page 9 last modified 2016-01-20: z correction is 1.32mm/20C
+    - [0]
+    - [0]
+    - [0, 66.0]
```

Comment:
This should negative because when T goes higher we want to move M2 and the camera closer to M1M3.

Reply:
I take it hexapod z points away from the primary? I was wondering (naively I would have guessed the opposite).

Reply:
yes z points away from the primary.